src/main/java/taxi/controller/GetAllCarsController.java

```diff
@@ -0,0 +1,27 @@
+package taxi.controller;
+
+import java.io.IOException;
+import java.util.List;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import taxi.lib.Injector;
+import taxi.model.Car;
+import taxi.service.CarService;
+
+public class GetAllCarsController extends HttpServlet {
+
+    private static final Injector injector = Injector.getInstance("taxi");
+    private CarService carService = (CarService) injector.getInstance(CarService.class);
+
+    @Override
+    protected void doGet(HttpServletRequest req, HttpServletResponse resp)
+            throws ServletException, IOException {
+
+        List<Car> allCars = this.carService.getAll();
+
+        req.setAttribute("cars", allCars);
```

Comment:
```suggestion
            throws ServletException, IOException {
        List<Car> allCars = carService.getAll();
        req.setAttribute("cars", allCars);
```
we don't need to use 'this' here

Reply:
done